src/commands/run-script.js

```diff
@@ -46,6 +46,7 @@ exports.handler = async ({ script, branch }, state) => {
 	return promiseAllErrors(
 		state.repos.map(async repository => {
 			const repoLabel = `${repository.owner}/${repository.name}`
+			const prefixedBranch = `nori/${branch}`
```

Comment:
it'd be good if we could add the prefix to the enquirer prompt so the user knows what to expect. i'm not sure what the best way of doing that is, i experimented with adding 

```js
format: value => `nori/${value}`
```

on line 32 but that seems to hide the cursor?

<img alt="nori branch prompt with nori prefix" src="https://user-images.githubusercontent.com/631757/95304272-474b0d80-087c-11eb-9d99-268ed14420a4.png">

Reply:
<img width="366" alt="Screen Shot 2020-10-07 at 9 41 25" src="https://user-images.githubusercontent.com/20328348/95346280-47014100-0881-11eb-987c-4505faaa8b87.png">

:)